src/asciidoc/docs/publish/customDomains.adoc

```diff
@@ -64,9 +61,39 @@ If you want to host your Koji on a subdomain of your custom domain name (for exa
 The domain name should be displayed.
 . Click the domain name.
 You should see a message verifying that your custom domain has been successfully assigned to the Koji.
+. Follow the steps for <<_configuring_dns_records, configuring DNS records>>.
 . Click *Open* to go to the new URL.
 Your Koji should be displayed.
 
+=== Configuring DNS records
+
+If you want to host your Koji on a subdomain of your custom domain name (for example, `+++www.mycustomdomain.com+++` or `+++game.mycustomdomain.com+++`), you will need to use your domain provider's tools to add a CNAME record for that subdomain (www or game) that points to `+++cnames.withkoji.com+++`.
+
+If you want your Koji to be served from the root domain name (`+++mycustomdomain.com+++`), you need to add an *A record* for the domain's root that maps it to Koji's *IP block*:
+
+151.101.1.78 +
+151.101.65.78 +
+151.101.129.78 +
+151.101.193.78 +
+
+If you are configuring a root domain via *A records*, it is also recommended to configure *AAAA records* to support *IPv6*.
+Koji's *IPv6 block* is:
+
+2a04:4e42::334 +
+2a04:4e42:200::334 +
+2a04:4e42:400::334 +
+2a04:4e42:600::334 +
+2a04:4e42:800::334 +
+2a04:4e42:a00::334 +
+2a04:4e42:c00::334 +
+2a04:4e42:e00::334 +
+
+Different domain providers have different tools for managing DNS records, and some of those tools are more complex than others.
+If you have questions about how to configure your domain's DNS records, please connect with us on *Discord* and someone will be happy to help.
+
+Managed subdomains (`+++withkoji.com+++`, `+++koji.sh+++`, and `+++koji.game+++`) automatically manage HTTPS termination.
```

Comment:
This information seems important but a bit hidden and/or out of place. Does this information belong in the Koji root domain section? Or, is it applicable to both sections?